sdk/batch/Microsoft.Azure.Batch/Tools/ObjectModelCodeGenerator/Spec/CloudPool.json

```diff
@@ -468,6 +468,17 @@
             "Name": "TargetNodeCommunicationMode"
           }
         },
+        {
+          "Key": {
+            "Type": "UpgradePolicy",
```

Comment:
I think the type should be `UpgradePolicy?` here

Reply:
Nevermind - looks like this follows our existing patterns for optional properties which are objects.